### PR TITLE
Add CircleCI configuration for building and releasing

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by geoand** [comment]
@cmoulliard @metacosm Seems like there are dependencies on the geneator. What do we need to do in this case?

**Comment by cmoulliard** [comment]
@geoand I will have a look now

**Comment by geoand** [comment]
OK, thanks

**Comment by cmoulliard** [comment]
As we don't have anymore dependency with `generator` project, then CircleICi build should work @geoand

**Comment by geoand** [comment]
Tests passing now, can you review please @cmoulliard ?

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    - run:
+        name: Perform build
+        command: |
+          make build
+
+  release:
+    docker:
+    - image: circleci/golang:1.9
+
+    working_directory: /go/src/github.com/snowdrop/generator
```
Comment (cmoulliard) [inline .circleci/config.yml]: Why do the directory points to `/go/src/github.com/snowdrop/generator` and not `/go/src/github.com/snowdrop/k8s-supervisor` ?

Comment (geoand) [reply .circleci/config.yml]: Good catch, that is bug (albeit with no actual significance). I'll fix it when I get around to updating the PR

**Comment by geoand** [comment]
Thanks for updating and merging the PR @cmoulliard

**Pull request merged**